db/migrate/20220821175150_create_users.rb

```diff
@@ -4,7 +4,6 @@ def change
       t.string :name
       t.string :email
       t.string :description
-
```

Comment:
Extra line was removed. You should not have changes in this file. Please, add empty line :)

Reply:
ohh)